pyproject.toml

```diff
@@ -100,6 +100,7 @@ ignore-init-module-imports = true
     "S605",  # todo: Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`
     "S607",  # todo: Starting a process with a partial executable path
     "RET504",  # todo:Unnecessary variable assignment before `return` statement
+    "RET503",
```

Comment:
Had to add this cc @Borda. Otherwise, pre-commit would force format the code with `return None` at the end.

Reply:
Maybe raise the issue with Ruff? :thinking: